apps/store/src/app/[locale]/checkout/components/CheckoutForm/CheckoutForm.tsx

```diff
@@ -77,64 +75,46 @@ export function CheckoutForm({
   })
 
   return (
-    <>
-      <form id={SIGN_FORM_ID} onSubmit={handleSubmitSign}>
-        <div className={yStack({ gap: 'xxs' })}>
-          <PersonalNumberField
-            label={t('FIELD_PERSONAL_NUMBER_SE_LABEL')}
-            value={ssn}
-            readOnly
-            disabled
-          />
-          {shouldCollectName && (
-            <>
-              <TextField
-                type="text"
-                label={t('FORM_FIRST_NAME_LABEL')}
-                name={FormElement.FirstName}
-                required
-              />
-              <TextField
-                type="text"
-                label={t('FORM_LAST_NAME_LABEL')}
-                name={FormElement.LastName}
-                required
-              />
-            </>
-          )}
-          {shouldCollectEmail && (
+    <form id={SIGN_FORM_ID} onSubmit={handleSubmitSign}>
+      <div className={yStack({ gap: 'xxs' })}>
```

Comment:
Seems to be 8px in design 
```suggestion
      <div className={yStack({ gap: 'xs' })}>
```

Reply:
We've done a lot of back and fourth on the design and spacings and I think the design fell out of sync.